ocdskingfisherprocess/web/app.py

```diff
@@ -77,14 +77,24 @@ def api_v1_submit_file():
     file_data_type = request.form.get('data_type')
     file_encoding = request.form.get('encoding', 'utf-8')
 
-    (tmp_file, tmp_filename) = tempfile.mkstemp(prefix="ocdskf-")
-    os.close(tmp_file)
+    if 'file' in request.files:
 
-    request.files['file'].save(tmp_filename)
+        (tmp_file, tmp_filename) = tempfile.mkstemp(prefix="ocdskf-")
+        os.close(tmp_file)
 
-    store.store_file_from_local(file_filename, file_url, file_data_type, file_encoding, tmp_filename)
+        request.files['file'].save(tmp_filename)
 
-    os.remove(tmp_filename)
+        store.store_file_from_local(file_filename, file_url, file_data_type, file_encoding, tmp_filename)
+
```

Comment:
Not sure whey there is a blank line between these statements.

Reply:
I'll try to make the layout a bit better - thanks